pages/common/gnmic.md

```diff
@@ -0,0 +1,24 @@
+# gnmic
+
+> A gNMI CLI client that provides full support for Capabilities, Get, Set and Subscribe RPCs with collector capabilities.
+> More information: <https://gnmic.kmrd.dev>.
```

Comment:
Taking the perspective of someone that's not familiar with `gnmic` or gNMI more broadly, I'd want this description to explain what `gnmic` is used for and why I might use it. This description indicates what gnmic is capable of but if I'm a new user that just encountered this command in a script and want to figure out what it's doing, I won't gain that understanding beyond knowing that I need to start searching for "gNMI".

This [Cisco gNMI Protocol](https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/prog/configuration/168/b_168_programmability_cg/gNMI_protocol.pdf) document indicates that gNMI is used to interact with/set configuration on network devices. Would the following be an accurate description of what gnmic is used for?

```suggestion
> A gNMI command-line client.
> Manage gNMI network device configuration and view operational data.
> More information: <https://gnmic.kmrd.dev>.
```